### DISCUSSION TITLE
Add manual customization of combo colors mid-map

### DISCUSSION THREAD

**Discussion by lunauii** [ppy/osu#25572] [Ideas] [answered]
This is a parity feature request from stable to lazer.

Stable currently allows you to manually set the combo color of a combo in a beatmap by expanding the New Combo arrow after selecting the first note in a combo:
![image](https://github.com/ppy/osu/assets/65362444/a06bb8d2-fa49-48d0-9396-faa6bd4c6fe4)

In lazer, this currently does not exist or is not in an intuitive position:
![image](https://github.com/ppy/osu/assets/65362444/1e299eb1-d235-4647-a274-e9ddca60c16e)

This is currently preventing lazer mappers from adding really creative thematic combo sections and instead makes us rely on implementing an excess amount of colors in a map to force a combo color in a specific section.

I'd like there to be a menu toggle when right clicking a combo starter:
![image](https://github.com/ppy/osu/assets/65362444/01becd81-2e78-4d7f-8619-cd902f4c8e52)

or perhaps even a new option in the inspector for combo starters:
![image](https://github.com/ppy/osu/assets/65362444/22edf51f-3f77-4dbe-8326-5054304645f4)

It would really help if this would be implemented and would make the laser editor even closer to stable! :pray:

**Comment by peppy** [comment, marked as answer]
Tracking at https://github.com/ppy/osu/issues/25608.
